model/vsstree.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 
 #
-#
+# (C) 2021 Robert Bosch GmbH
```

Comment:
Correct to add this when modifying code.

In addition, it looks like Felix Willnecker committed most of this new vsstree.py module when it was rewritten.  Should this be complemented by adding BMW's copyright line? (@danielwilms ?)

Note:  There are some not insignificant changes by Felix also in vspec.py